bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md

```diff
@@ -0,0 +1,66 @@
+#  Connecting Amazon OpenSearch Service to Microsoft Power BI Desktop
+
+>[!Note]
+
+>The following connector article is provided by Amazon, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the OpenSearch website and use the support channels there.
+
+## Summary
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
+| Products | Power BI Desktop |
+| Authentication Types Supported | Basic |
+
+## Prerequisites
+* Microsoft Power BI Desktop
+* [OpenSearch](https://docs-beta.opensearch.org/opensearch/install/index/)
+* [OpenSearch SQL ODBC driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/)
+
+## Capabilities supported
+* Import
+* DirectQuery
+
+## Connect to Amazon OpenSearch Service
+1. Open Power BI Desktop.
+
+2. Click on **Home** > **Get Data** > **More** > **Other**. Select **Amazon OpenSearch Service**. Click on **Connect**.
+
+<img src="img/pbi_select_connector_amz.png" width="500">
+
+3. You will get a warning for using third-party service. Click on **Continue**.
+
+<img src="img/pbi_third_party_warning.png" width="500">
+
+4. Enter server value. Click on **OK**.

```

Comment:
```suggestion
4. Enter the server value. Click on **OK**.

```

Reply:
What is the server value? The hostname of the OpenSearch service?

Reply:
This should be updated. This was copied from the [old markdown file](https://github.com/Bit-Quill/opensearch-project-sql/blob/main/sql-odbc/docs/user/power_bi_support.md#:~:text=Enter%20server%20value.%20Click%20on%20OK.). I will update both files